### DISCUSSION TITLE
What Lora Format is this? and how to convert it to diffusers format?

### DISCUSSION THREAD

**Discussion by nighting0le01** [huggingface/diffusers#7552] [General]
Hi i came accross this lora format for a RV5.1 (SD 1.5 based model) dict_keys(['hookers.layers.0.temporal_i.bias', 'hookers.layers.0.temporal_i.weight', 'hookers.layers.0.temporal_k.bias', 'hookers.layers.0.temporal_k.weight', 'hookers.layers.0.temporal_n.bias', 'hookers.layers.0.temporal_n.weight', 'hookers.layers.0.temporal_o.bias', 'hookers.layers.0.te..
this is from the https://huggingface.co/LayerDiffusion/layerdiffusion-v1/tree/main https://huggingface.co/LayerDiffusion/layerdiffusion-v1/resolve/main/layer_sd15_transparent_attn.safetensors?download=true
how to convert and load this into SD1.5 based models

**Comment by sayakpaul** [comment]
Are you sure it's LoRA?

**Reply by nighting0le01** [reply]
@sayakpaul Hi yes it is : it's from LayerDiffusion/layerdiffusion-v1, https://huggingface.co/LayerDiffusion/layerdiffusion-v1/tree/main,  
here is the exact LoRA weight: https://huggingface.co/LayerDiffusion/layerdiffusion-v1/resolve/main/layer_sd15_transparent_attn.safetensors?download=true. 

orignal repo reference:https://github.com/layerdiffusion/sd-forge-layerdiffuse#:~:text=layer_sd15_transparent_attn.safetensors

**Reply by sayakpaul** [reply]
Indeed. I think they have additional sources that would be needed to complement such a LoRA checkpoint. More specifically:

* https://github.com/layerdiffusion/sd-forge-layerdiffuse/blob/e4d5060e05c7b4337a3258bb03c4e3ad2f8b15bb/lib_layerdiffusion/models.py#L52
* https://github.com/layerdiffusion/sd-forge-layerdiffuse/blob/main/lib_layerdiffusion/attention_sharing.py

**Reply by nighting0le01** [reply]
@sayakpaul  But those seem to be in LDM format. anyway to apply to a diffusers SD1.5 checkpoint? or get a diffusers implementation

**Reply by sayakpaul** [reply]
Not yet
